content/sensu-go/5.16/api/roles.md

```diff
@@ -125,7 +194,8 @@ The `/roles/:role` API endpoint provides HTTP GET access to [role data][1] for s
 In the following example, querying the `/roles/:role` API endpoint returns a JSON map that contains the requested [`:role` definition][1] (in this example, for the `:role` named `read-only`).
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/roles/read-only -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/core/v2/namespaces/default/roles/read-only
```

Comment:
requires ' \ ' at the end of the line, probably should move 'curl -X GET' to it's own line for consistency